packages/search-engine/src/direflow-components/search-engine/App.tsx

```diff
@@ -1,34 +1,121 @@
-import { Styled } from 'direflow-component';
 import React, { FC } from 'react';
-import { useFetch } from '../../hooks/use-fetch';
-import styles from './App.css';
+import './App.css';
+import { ReactiveBase, DataSearch, DateRange, MultiDataList } from '@appbaseio/reactivesearch';
 
-export interface Tweet {
-  id: number
-  text: string
-}
 
 const App: FC = () => {
-  const [tweets, setTweets] = React.useState<Tweet[]>();
-  const { fetchApi } = useFetch();
+  return (
+    <ReactiveBase
+          app="search-engine"
+          url="http://twint.f4.htw-berlin.de:9200/"
+          enableAppbase
+          
 
-  React.useEffect(() => {
-    fetchApi('api/search-tweets')
-      .then((res) => res.json())
-      .then((json) => setTweets(json.tweets));
-  }, []);
+          theme={{
+            typography: {
+               fontFamily: '-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Noto Sans", "Ubuntu", "Droid Sans", "Helvetica Neue", sans-serif',fontSize: "16px"
+            },
+            colors: {
+                  textColor: "#fff",
+                  backgroundColor: "#212121",
+                  primaryTextColor: "#fff",
+                  primaryColor: "#2196F3",
+                  titleColor: "#fff",
+                  alertColor: "#d9534f",
+                  borderColor: "#666"
+            }
+          }}
+    >
+      {/*Search-Bar*/}
+      <div className="navbar">
+        
+        <div className="logo-container">
+          //some logo here
+        </div>
+         
+        
 
-  return (
-    <Styled styles={styles}>
-      <div className="app">
-        {tweets && tweets.map((tweet) => (
-          <div key={tweet.id}>
-            <span style={{ marginRight: '1rem' }}>{tweet.id}</span>
-            <span>{tweet.text}</span>
+        <div className="search-container">
+          <DataSearch            
+            componentId="mainSearch"            
+            dataField={["party","party.search", "tweets", "tweets.search"]}                      
+            className="search-bar"            
+            queryFormat="and"            
+            placeholder="Search here..."
+            innerClass={{
+              "input": "searchbox",
+              "list": "suggestionlist"
+            }}                  
+          />
+        </div>
+      </div>
+      
+      
+      {/*Filters*/}
+      <div className="left-bar">
+        <div>
+          <div className="filter-heading center">
+            <b>
+              {" "}
+              <i className="fa fa-calendar" /> Party Filter{" "}
+            </b>
+          </div> 
+
+          <MultiDataList
+              componentId="party-list"
+              dataField="party"
+              data={[
```

Comment:
Maybe move this into a variable, so that it's easier to change this?

Reply:
Also, list item is not readable (text color)
![image](https://user-images.githubusercontent.com/61015467/144751880-58f7f26a-154c-4411-9838-614eb607298d.png)